src/main/java/com/minecolonies/coremod/colony/permissions/Permissions.java

```diff
@@ -718,7 +718,7 @@ public boolean addPlayer(@NotNull final String player, final Rank rank, final Wo
         }
         final GameProfile gameprofile = world.getMinecraftServer().getPlayerProfileCache().getGameProfileForUsername(player);
         //Check if the player already exists so that their rank isn't overridden
-        return gameprofile != null && !players.containsKey(gameprofile.getId()) && addPlayer(gameprofile, rank);
+        return gameprofile != null && !getOwner().equals(gameprofile.getId()) && addPlayer(gameprofile, rank);
```

Comment:
Don't merge this yet, this creates a small security issue. I'll look for the best solution tonight.

Allows Officers to demote eachother.